configure.ac

```diff
@@ -2444,8 +2434,31 @@ fi
         echo "       for building the distribution"
         echo "   To install: cargo install cargo-vendor"
       fi
+      AC_CHECK_FILES([$srcdir/rust/gen], [have_rust_headers="yes"])
```

Comment:
if we do have the headers already, doesn't that mean we can skip the block directly below?